composer.json

```diff
@@ -0,0 +1,5 @@
+{
+    "require": {
+        "pear/console_commandline": "^1.2"
```

Comment:
I had doubts about adding a dependency but then I thought - I don't see a point in reinventing a wheel. The libraries are out there for a reason - you don't have to redo what has already been done. The alternative is to create another command line parser which I don't like. I believe the composer is a way to work with dependencies and I don't see it to be a problem that someone doesn't use it in his project. It's fairly simple to use and because the `protoc-gen-php` is external to your project it does't mean now you need to use composer in it too.